.codecov.yml

```diff
@@ -0,0 +1,25 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "20...100"
```

Comment:
I think the second number set to 100 doesn't make much sense when reading https://docs.codecov.com/docs/coverage-configuration#range. I would go with 70 or 75

Reply:
@tremes check if DVO doesn’t have the same config. If I’m not wrong I copied the same config, maybe we want to change it there as well.